build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+apply plugin: 'java'
+apply plugin: 'maven-publish'
+apply plugin: 'idea'
+apply plugin: 'war'
+
```

Comment:
You can use allprojects {} or subProjects {} to avoid duplicating importing dependencies